### ISSUE TITLE
Shown stack dumps in auth:regenerated

### ISSUE THREAD

**Issue by monaka** [deis/workflow-cli#219]
Possibly this is caused by an authentication trouble on my instance.
I think it's better messages were organized.

```
$ deis auth:regenerate --all 
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x55d9a1]

goroutine 1 [running]:
panic(0x87ffc0, 0xc8200100e0)
        /usr/local/go/src/runtime/panic.go:481 +0x3e6
github.com/deis/workflow-cli/vendor/github.com/deis/controller-sdk-go/auth.Regenerate.func1(0x0)
        /go/src/github.com/deis/workflow-cli/vendor/github.com/deis/controller-sdk-go/auth/auth.go:110 +0xb1
github.com/deis/workflow-cli/vendor/github.com/deis/controller-sdk-go/auth.Regenerate(0xc820055080, 0x0, 0x0, 0x1, 0x0, 0x0, 0x0, 0x0)
        /go/src/github.com/deis/workflow-cli/vendor/github.com/deis/controller-sdk-go/auth/auth.go:115 +0x2d7
github.com/deis/workflow-cli/cmd.Regenerate(0x0, 0x0, 0xc8200d9901, 0x0, 0x0)
        /go/src/github.com/deis/workflow-cli/cmd/auth.go:279 +0xa4
github.com/deis/workflow-cli/parser.authRegenerate(0xc82000a1c0, 0x2, 0x2, 0x0, 0x0)
        /go/src/github.com/deis/workflow-cli/parser/auth.go:238 +0x290
github.com/deis/workflow-cli/parser.Auth(0xc82000a1c0, 0x2, 0x2, 0x0, 0x0)
        /go/src/github.com/deis/workflow-cli/parser/auth.go:40 +0x686
main.Command(0xc82000a1c0, 0x2, 0x2, 0x59df15)
        /go/src/github.com/deis/workflow-cli/deis.go:92 +0xe3b
main.main()
        /go/src/github.com/deis/workflow-cli/deis.go:18 +0x60
```

```
$ deis --version
2.3.0-2ce1908
```

**Comment by monaka** [comment]
Almost same result on v2.4.0 also.

```
./deis auth:regenerate --all 
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x5521d1]

goroutine 1 [running]:
panic(0x8ef380, 0xc8200100e0)
        /usr/local/go/src/runtime/panic.go:481 +0x3e6
github.com/deis/workflow-cli/vendor/github.com/deis/controller-sdk-go/auth.Regenerate.func1(0x0)
        /go/src/github.com/deis/workflow-cli/vendor/github.com/deis/controller-sdk-go/auth/auth.go:110 +0xb1
github.com/deis/workflow-cli/vendor/github.com/deis/controller-sdk-go/auth.Regenerate(0xc8201dd110, 0x0, 0x0, 0x1, 0x0, 0x0, 0x0, 0x0)
        /go/src/github.com/deis/workflow-cli/vendor/github.com/deis/controller-sdk-go/auth/auth.go:115 +0x2d7
github.com/deis/workflow-cli/cmd.DeisCmd.Regenerate(0x0, 0x0, 0x0, 0x0, 0xc820133b01, 0x0, 0x0)
        /go/src/github.com/deis/workflow-cli/cmd/auth.go:280 +0xbd
github.com/deis/workflow-cli/cmd.(*DeisCmd).Regenerate(0xc8200f5a70, 0x0, 0x0, 0xc820133b01, 0x0, 0x0)
        <autogenerated>:15 +0xc1
github.com/deis/workflow-cli/parser.authRegenerate(0xc82018c740, 0x2, 0x2, 0x7f5dc2769920, 0xc8200f5a70, 0x0, 0x0)
        /go/src/github.com/deis/workflow-cli/parser/auth.go:241 +0x2a9
github.com/deis/workflow-cli/parser.Auth(0xc82018c740, 0x2, 0x2, 0x7f5dc2769920, 0xc8200f5a70, 0x0, 0x0)
        /go/src/github.com/deis/workflow-cli/parser/auth.go:40 +0x743
main.Command(0xc82018c740, 0x2, 0x2, 0x24)
        /go/src/github.com/deis/workflow-cli/deis.go:104 +0x10e0
main.main()
        /go/src/github.com/deis/workflow-cli/deis.go:19 +0x60
```

```
$ ./deis --version
v2.4.0
```

**Comment by helgi** [comment]
@monaka This was fixed a few days ago and will be part of 2.5 that comes out this week. see https://github.com/deis/controller-sdk-go/pull/84
